src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -220,22 +228,38 @@ private long createNewCaseInCcd(
             idamToken,
             s2sToken,
             userId,
-            jurisdiction,
+            exceptionRecord.poBoxJurisdiction,
             caseCreationDetails.caseTypeId,
             true,
             CaseDataContent
                 .builder()
-                .caseReference(originalCaseId)
+                .caseReference(exceptionRecord.id)
                 .data(caseCreationDetails.caseData)
                 .event(Event
                     .builder()
                     .id(eventResponse.getEventId())
                     .summary("Case created")
-                    .description("Case created from exception record ref " + originalCaseId)
+                    .description("Case created from exception record ref " + exceptionRecord.id)
                     .build()
                 )
                 .eventToken(eventResponse.getToken())
                 .build()
         ).getId();
     }
+
+    private void checkBulkScanReferenceIsSet(Map<String, ?> caseData, String exceptionRecordId) {
```

Comment:
Can we not set bulk scan reference and not rely on services ?

Reply:
Also we would need to update phase 2 case field definition in Confluence so that doesn't get missed

Reply:
absolutely. that's my next step

Reply:
Sorry, which one have you just agreed on - setting it ourselves or verifying if the transformation service provided it?

Reply:
@lgonczar  we agreed that we would not modify case data coming back from transformation. and if we send something which is not configured in ccd by them (services) then it’s our fault we broke the callback which we don't want to happen and hence logging would be better as it would make us aware.